ScratchDjango/User/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from .views import register_user, register_user_template
+
+urlpatterns = [
+    path("register_user/", register_user, name="register_user"),
+]
+template_url_patterns = [
+    path("register_user_template/", register_user_template, name="register_user_template"),
```

Comment:
I think the naming convention would be better this way `name="register-user-template"`.

Reply:
But in the best practices book, the author said that we can use -(hyphen) in urls but it's very wrong to use them in name